src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -290,6 +306,24 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
             }
         }
 
+        private void ExecuteClusterTransaction(TransactionOperationContext context, BlittableJsonReaderObject cmd, long index, out bool success)
```

Comment:
Avoid using out variable when you can use return code.
And using `bool` here doesn't help much. We need to return an explicit explanation of what failed if we weren't able to apply things.